src/components/NewCommentForm.tsx

```diff
@@ -99,19 +117,16 @@ export const NewCommentForm: React.FC<Props> = ({ onSubmit }) => {
             </span>
           )}
         </div>
-
         {errors.name && (
           <p className="help is-danger" data-cy="ErrorMessage">
             Name is required
           </p>
         )}
       </div>
-
       <div className="field" data-cy="EmailField">
         <label className="label" htmlFor="comment-author-email">
           Author Email
         </label>
-
         <div className="control has-icons-left has-icons-right">
           <input
             type="text"
```

Comment:
It seems that you are using a text type for an email input field. This could lead to validation issues as any string would be considered valid. It would be better to use 'email' type for email input field. It provides basic validation and more suitable keyboard layout on mobile devices.